crates/subspace-farmer-components/src/lib.rs

```diff
@@ -47,7 +47,7 @@ impl SectorMetadata {
     pub fn encoded_size() -> usize {
         let default = SectorMetadata {
             total_pieces: NonZeroU64::new(1).expect("1 is not 0; qed"),
-            expires_at: 0,
+            expires_at: SegmentIndex::default(),
```

Comment:
I would prefer `SegmentIndex::ZERO` as `Default`-s are not always clear

Reply:
In this particular context it makes no difference, we only create values to measure encoded size.